README.md

```diff
@@ -1,2 +1,442 @@
+
+e.	Save the configuration file.
+
+f.	Exit the text editor and restart the apache server from the XAMPP control panel.
+
+3.	Next, we need to install Drupal 7.x which can be found and downloaded from here https://www.drupal.org/project/drupal/releases/7.80. Please pick the latest Drupal 7.x version from the Other Releases sidebar on the website. Unpack the downloaded file under /xampp/htdocs/.
+
+4.	(Optional) Change the title of the unpacked folder to ‘drupal’ as an example.
+
+5.	Open up a web browser of your choice, type in the address bar ‘localhost:5555/phpmyadmin’ (5555 is the port number that you set in the httpd.conf file), and press enter.
+
+6.	In the PHPMyAdmin panel find and click the ‘Databases’ navigation menu button.
+
+7.	Under the ‘Create Database’ type in the database name e.g ‘drupal-h5p’ and click the ‘Create’ button.
+
+8.	Open up a web browser, type in ‘localhost:5555/drupal’, and press enter. You should be able to see the Drupal installation guide. If the guide displays an error message that says the extension ‘gd’ is not enabled:
+
+a.	Open the XAMPP control panel.
+
+b.	On the row that has Apache controls click ‘Config’ and choose the very first option ‘PHP (php.ini)’.
+
+c.	In that file find the line that starts with ‘;extension=gd2’ replace it with ‘extension=gd2’.
+
+d.	Save the file, reload the Apache server and visit the same drupal installation guide page ‘localhost:5555/drupal’.
+
+9.	If the Drupal installation guide displays a warning that says ‘PHP OPCODE CACHING’ you either have to install the displayed plugin or scroll down and click the ‘continue anyway’ option.
+
+10.	On the ‘Set up database’ step type in the name of the database that you created e.g ‘drupal-h5p’. In the ‘Database username’ field type in root and the password field is optional.
+
+11.	Wait for the Drupal installation to complete.
+
+12.	Fill in the details in the ‘Configure Site’ form and Drupal should be ready to go.
+
+13.	Find the latest version of H5P for Drupal 7.x here https://www.drupal.org/project/h5p e.g ‘7.x-1.48 released 22 April 2019’ and click download either in .zip or .tar.gz format.
+
+14.	Unpack the download archive file under /xampp/htdocs/drupal/modules.
+
+15.	In the web browser go back to the Drupal admin panel ‘localhost:5555/drupal/’.
+
+16.	Click the ‘Modules’ navigation button.
+
+17.	Scroll down on the modules page or find ‘H5P’ using the browser’s search function.
+
+18.	Select both ‘H5P’ and ‘H5P Editor’ and click the ‘Save Configuration’ button underneath.
+
+19.	Click the ‘Configure’ navigation button on the top navbar, then click on H5P under the system configuration group, scroll down, and enable both:
+
+a.	H5P development mode
+
+b.	Library development directory (for programmers only)
+
+20.	If everything is working correctly you should see the Interactive Content type when you click Add Content from the top menu.
+
+
+
+
+
+1. Got to the (C:\xampp\htdocs\drupal\sites\default\files\h5p\development),
```

Comment:
Were these supposed to fall under another header that got removed...? "Cloning the project" perhaps?

Reply:
Or perhaps they just got renumbered and should be 21, 22, etc.?